File: 2076-sum-of-digits-of-string-after-convert/2076-sum-of-digits-of-string-after-convert.cpp
class Solution {
public:
    int getLucky(string s, int k) {
        string ascii;

        for (int i = 0; i < s.size(); i++){
            ascii.append(to_string(static_cast<int>(s[i] - 'a' + 1)));
        }
        
        for (int i = 0; i < k ; i++){
            int sum = 0;
            for (int j = 0; j < ascii.size(); j++){
                sum += ascii[j] - '0'; 
            }
            ascii = to_string(sum);
        }
        
        return stoi(ascii); 
    }
};

File: 2076-sum-of-digits-of-string-after-convert/2076-sum-of-digits-of-string-after-convert.py
class Solution:
    def getLucky(self, s: str, k: int) -> int:
        asc = ""

        for i in range(len(s)):
            asc += str(ord(s[i]) - 96)
        
        for i in range(k):
            sumx = 0
            for j in range(len(asc)):
                sumx += int(asc[j])
            asc = str(sumx)

        return int(asc)